src/otk_external_osbuild/command/get_dnf4_package_info.py

```diff
@@ -0,0 +1,39 @@
+import json
+import sys
+from typing import List, Optional, TextIO
+
+
+def find_pkg_by_name(packages: List[dict], pkg_name: str) -> Optional[dict]:
+    for pkg in packages:
+        if pkg["name"] == pkg_name:
+            return pkg
+    return None
+
+
+def root(input_stream: TextIO) -> None:
+    data = json.load(input_stream)
+    pkg_name = data["tree"]["package_name"]
```

Comment:
This is going to be EXTREME NITPICKING but `packagename` to be consistent with `packageset`?

Reply:
It does look nicer/more consistent this way, thanks you! I updated the PR to use "packagename" now.